backend/sample.env

```diff
@@ -89,9 +89,9 @@ PROMPT_PORT=3003
 PROMPT_STUDIO_FILE_PATH=/app/prompt-studio-data
 
 # Structure Tool
-STRUCTURE_TOOL_IMAGE_URL="docker:unstract/tool-structure:0.0.20"
+STRUCTURE_TOOL_IMAGE_URL="docker:unstract/tool-structure:0.0.21"
 STRUCTURE_TOOL_IMAGE_NAME="unstract/tool-structure"
-STRUCTURE_TOOL_IMAGE_TAG="0.0.20"
+STRUCTURE_TOOL_IMAGE_TAG="0.0.21"
```

Comment:
@harini-venkataraman Which version of SDK does this build refer to?

Reply:
Pls make sure the toools are pinned to 0.26.0

Reply:
@gaya3-zipstack This change is not needed now since SDK tool version have no changes. This is reverted. Missed in the downgrade.